### PR TITLE
[PLAY-91] Add unique key prop to text input

### PR DESCRIPTION
**Pull request by kyleschechter**
#### Screens

image

#### Breaking Changes

No. A unique key prop was added to the actual input element of the Text Input kit, clearing the error message that usually persists in the console when working with this kit in React.

#### Runway Ticket URL

[https://nitro.powerhrg.com/runway/backlog_items/PLAY-91](url)

#### How to test this

1. Navigate to http://localhost:3000/kits/text_input/react
2. Open console log
3. Check to make sure error is not there anymore

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

File: playbook/app/pb_kits/playbook/pb_text_input/_text_input.jsx [hunk 1]
```diff
@@ -89,6 +89,7 @@ const TextInput = (props: TextInputProps, ref: React.ElementRef<"input">) => {
         className="text_input"
         disabled={disabled}
         id={id}
+        key={`${id}_${className}`}
```
Comment (JelenaD1) [inline playbook/app/pb_kits/playbook/pb_text_input/_text_input.jsx:92]: Will this work with assigning `id` only? Just curious what className will do here?

Comment (kyleschechter) [reply playbook/app/pb_kits/playbook/pb_text_input/_text_input.jsx:92]: it was acting weird when i used only the id, so i added the classname just to be safe. Just to ensure uniqueness pretty much

**Review by thestephenmarshall — CHANGES_REQUESTED** [review]
After thinking about this more, if [there is not a default](https://github.com/powerhome/playbook/blob/39d7c754f1db9aaaff23540595e890cf3f306df5/playbook/app/pb_kits/playbook/pb_text_input/_text_input.jsx#L46) then the key will always be the _`className` prop string which will not be unique.

**Comment by kyleschechter** [comment]
@thestephenmarshall Is there ever going to be a scenario where there isn't an 'id'?  I just figured there would always be one. If not though, I can look into a function or something to generate unique id's.

**Comment by thestephenmarshall** [comment]
@kyleschechter of course there could be. [Here](https://github.com/powerhome/playbook/blob/39d7c754f1db9aaaff23540595e890cf3f306df5/playbook/app/pb_kits/playbook/pb_text_input/_text_input.jsx#L21) you have an optional prop which means it can be `undefined`. And [here](https://github.com/powerhome/playbook/blob/39d7c754f1db9aaaff23540595e890cf3f306df5/playbook/app/pb_kits/playbook/pb_text_input/_text_input.jsx#L46) you have no default value.

**Review by thestephenmarshall — APPROVED** [review]

**Pull request merged**